### ISSUE TITLE
Action fails with "fail to parse diff" error if PR touches more than 300 files.

### ISSUE THREAD

**Issue by menzenski** [reviewdog/action-actionlint#124]
We're using this action as a pull request check in a repository that sees a lot of autogenerated PRs touching many configuration files. If more than 300 files are changed by a PR, this action fails with:

```
reviewdog: post failed for actionlint: fail to parse diff: GET https://api.github.com/repos/<ORG>/<REPO>/pulls/380: 406 Sorry, the diff exceeded the maximum number of files (300). Consider using 'List pull requests files' API or locally cloning the repository instead. [{Resource:PullRequest Field:diff Code:too_large Message:}]
```

Is there a workaround here?

**Comment by colatkinson** [comment]
It looks like there was a partial fix in 0.17.4 (issue: https://github.com/reviewdog/reviewdog/issues/1696, PR: https://github.com/reviewdog/reviewdog/pull/1714). That said, the bump PR is still outstanding (https://github.com/reviewdog/action-actionlint/pull/123), and based on the issue comments it appears that there are still some unhandled cases.

That said hopefully a maintainer/someone affiliated with the project can correct me if I'm wrong here, I was just looking into this because I also hit this issue.

**Comment by massongit** [comment]
I applied reviewdog v0.17.5 include https://github.com/reviewdog/reviewdog/pull/1722 and release reviewdog/action-actionlint.
Therefore, I think this problem has been resolved.
